samples/matter/template/boards/nrf54l15pdk_nrf54l15_cpuapp_ns.overlay

```diff
@@ -37,3 +37,13 @@
 &wdt31 {
 	status = "okay";
 };
+
+// Change IRQ ids to handle button interrupts.
```

Comment:
Are we aware of any other possible interrupts that could be affected as well?

Reply:
Not yet, let's see :)